plugins/frontend/backstage-plugin-shortcut/README.md

```diff
@@ -0,0 +1,70 @@
+# Shortcut Plugin for Backstage
+
+![a preview of the Shortcut plugin](./docs/shortcut.png)
+
+This plugin provides overview of user stories which are not yet completed but are assigned to a user. In order to retrieve stories you will need to sign in to Backstage using same email address used for Shortcut account.
+
+## Rate Limit
+
+The Shortcut REST API limits requests to 200 per minute. Any requests over that limit will not be processed, and will return a 429 (“Too Many Requests”) response code.
+
+## Authentication
+
+The Shortcut API uses token-based authentication so in order to retrieve results you will need it. To generate an API token, go to https://app.shortcut.com/settings/account/api-tokens.
+
+## Features
+
+- Shortcut stories progress overview.
+
+## How to add shortcut plugin to Backstage app:
+
+If you have your own backstage application without this plugin, here it's how to add it:
+
+1. Install the plugin inside `backstage/packages/app` :
+
+```bash
+cd packages/app
+yarn add @roadiehq/backstage-plugin-shortcut
+```
+
+2. In the `app-config.yaml` file in the root directory, add shortcut to the proxy object:
+
+```yml
+proxy:
+  ...
+  '/shortcut/api':
+    target: https://api.app.shortcut.com/api/v3
+    headers:
+      Shortcut-Token: '${SHORTCUT_API_TOKEN}'
+```
+
+3. Add plugin to your Backstage instance:
+
+```ts
+// packages/app/src/components/catalog/EntityPage.tsx
+import { EntityShorcutPageContent } from '@roadiehq/backstage-plugin-shortcut';
+...
+
+const serviceEntityPage = (
```

Comment:
I think this plugin works for a user, not strictly on a per service level. I'd not add this here. I think we should encourage the use of the homepage plugin and /or using a full page that is accessible from the sidebar for it

Reply:
For the encouragement for the  usage of homepage plugin, I would still like to have this option open for OSS community in a way they would use it how it benefits them. That is one of the reasons I decided to go with creating a StoriesCard in ShortcutPage.

That way it can also be used in EntityPage as a widget. However, I do agree regarding service level, so definitely could move it there.

Reply:
Yea, make sense, let's meet in the middle and add some example to include it in their homepage!